ui/apps/platform/src/Containers/Collections/CollectionForm.tsx

```diff
@@ -257,6 +257,7 @@ function CollectionForm({
                         <>
                             <p>Extend this collection by attaching other sets.</p>
                             <CollectionAttacher
+                                collectionId={action.type === 'create' ? null : action.collectionId}
```

Comment:
For edge case for clone action, is the existing collection allowed or disallowed in the list?

Reply:
Aha great find. Yep this is an oversight, the clone action should allow all existing collections in the list.

Reply:
Fixed! I also update the parameter name to be a little bit more descriptive over what the ID is used for.